rust/blockstore/src/arrow/block/delta/delta.rs

```diff
@@ -72,8 +79,8 @@ impl BlockDelta {
     /// A tuple containing the the key of the split point and the new block delta.
     /// The new block deltas contains all the key value pairs after, but not including the
     /// split point.
-    pub(crate) fn split<'referred_data, K: ArrowWriteableKey, V: ArrowWriteableValue>(
-        &'referred_data self,
```

Comment:
Does clippy prefer not showing lifetimes when its trivially eliased? I somewhat prefer making things explicit but am fine with this if we deem it what we want.

Reply:
Oh nvm, its not used in the output I see